src/fork.rs

```diff
@@ -544,14 +545,14 @@ impl ForkDetails {
             estimate_gas_price_scale_factor,
             estimate_gas_scale_factor,
             fee_params,
-            cache_config,
+            cache_config: cache_config.clone(), // todo revisit
```

Comment:
Nit: Probably worth either a better explanation of task number reference, so that when it's discovered in X months, the person would understand the intent.